### PR TITLE
rename pool to batch

### PR DESCRIPTION
**Pull request by ToonRosseel**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-cmgg/exomecnv/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -29,63 +29,43 @@ If CRAM/CRAI files were provided in the samplesheet, they will be converted into
 <details markdown="1">
 <summary>Output files</summary>
 
-- `cram/`
+- `bam/`
   - `<sample>.bam`: BAM file
   - `<sample>.bam.bai`: BAI index file
   </details>
```
Comment (nvnieuwk) [inline docs/output.md:35]: I don't think this in the output (it was disabled before)

**Review by nvnieuwk — APPROVED** [review]
Just one small remark

File: docs/output.md (hunk 1 above)
Comment (ToonRosseel) [reply docs/output.md:35]: Correct. But you can enable it if you would like. I will leave it in for now

Comment (nvnieuwk) [reply docs/output.md:35]: I don't think so? The publishDir was commented out so I removed it in the last PR, I can re-add it if you want

**Pull request merged**